docs/operator/getting-started.md

```diff
@@ -97,6 +97,7 @@ rules:
   resources:
   - secrets
   - services
+  - configmaps
```

Comment:
The operator's ClusterRole should've always been configured to watch for configmaps but wasn't. cc @hjet for the helm charts.